### DISCUSSION TITLE
Feature Request: Brush Size Memory

### DISCUSSION THREAD

**Discussion by cirquesbreeze** [keiwando/toonsquid#344] [Ideas]
It would be fantastic if @keiwando could implement a "Brush Size Memory" feature in ToonSquid, similar to what is available in Procreate.

![IMG_8007](https://github.com/keiwando/toonsquid/assets/126395052/cd32a90a-71b3-4af3-b4b9-620f4fe4f524)

This feature would allow us to save and recall our preferred brush sizes, giving us more control and efficiency in the creative process. 

Currently, the brush size slider in ToonSquid has a fast inertia, making it challenging to select precise sizes. The slider often slips to nearby sizes. By incorporating the Brush Size Memory, it would at least in part mitigate this problem and provide us with greater accuracy and ease when going back and forth selecting specific brush sizes. 

Adding this functionality to the app could help to focus more on our creativity and less on struggling with getting brush sizes right.

**Comment by Cat-Butler** [comment]
Upvoted, I agree it would be very useful.

Note: If you drag your Pencil or finger vertically down away from the brush size slider (while still holding it down), you get finer control of the slider values, similar to Procreate. (Not currently documented in the [ToonSquid handbook](https://toonsquid.com/handbook/).)

You probably already realise:
Procreate only recently added those brush size "bookmarks". As an interim workaround in ToonSquid you can do the same as the old Procreate workaround, duplicate a brush and change the size settings to have multiple versions of the brush at different pre-set sizes.
